TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as

```diff
@@ -19,12 +19,20 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	{
 		case Hitters::builder:
 			dmg *= 2.00f;
+			if(isClient())
```

Comment:
same here

Reply:
darn i copied the isClient() from base kag